src/components/loader/loader.md

```diff
@@ -0,0 +1,13 @@
+Spinner loader
```

Comment:
Let's add documentation around the a11y once we figure out how to apply it to our bigmaven use-case. For example, we need to figure out how to use `aria-describedby` on the region that is loading. In addition, that region might also need an `aria-busy`.

We also need to investigate whether we need to allow the developer to customize the screen reader value: is "loading..." sufficient to visually impaired users?